src/Round.js

```diff
@@ -0,0 +1,42 @@
+const Turn = require("./Turn");
+
+class Round {
+    constructor(deck) {
+        this.playingDeck = deck.cards;
+        this.turn;
+        this.turnsCount = 0;
+        this.currentCard = deck.cards[0];
+
+        this.incorrectGuesses = [];
+        this.correctGuesses = [];
+    }
+
+    returnCurrentCard() {
+        return this.currentCard;
+    }
+
+    takeTurn(guess) {
+    
+        let currentPlayingCard = this.playingDeck;
+        this.turnsCount++;
+        this.currentCard = currentPlayingCard[this.turnsCount];
+
+        this.turn = new Turn(guess, this.currentCard)
+        let answer = this.turn.evaluateGuess()
+
+        if(!answer) {
```

Comment:
I really like how this condition is written, and I wonder if I can find a way to write this as a switch, instead.